### ISSUE TITLE
On boarding experience for new users/contributors is poor/confusing

### ISSUE THREAD

**Issue by bhouston** [gkjohnson/three-gpu-pathtracer#218]
**Is your feature request related to a problem? Please describe.**

We should aim to make it as easy as possible to get started with three-gpu-pathtracer.  Both in terms of getting it running after checking it out locally and also using it as a library in an external project.  I've had some novice developers totally get stuck trying to run three-gpu-pathtracer after checking it out.  The steps to get it working are not clear.

Specifically they get this error:

![Screenshot 2022-07-22 at 9 01 04 AM](https://user-images.githubusercontent.com/588541/180444534-f3fdf003-601b-4d56-86f7-7b3d024962d0.png)

**Describe the solution you'd like**

I would like it if we could have a single ``npm run dev`` command that does the following:
(1) start rollup on the library in a watch mode.
(2) start rollup on the examples in watch mode
(3) starts a web browser that auto-refreshes on changes.

This would allow a user to download this project, run ``npm install``, and then ``npm run dev`` and be presented with a development environment where they can make rapid changes.

This is near equivalent to ThreeJS's current ``npm run dev`` setup: https://github.com/mrdoob/three.js/blob/dev/package.json#L106

**Comment by bhouston** [comment]
Oh, I found it.  In CONTRIBUTING.md there is this:
```
## Developer setup

To develop and test changes to this library, make sure you have Node and NPM installed.
Check the supported versions in [the test configuration](./.github/workflows/node.js.yml).

In order to install dependencies, you will need `make` and a C++ compiler available.
On Debian or Ubuntu, run `sudo apt install build-essential`.

- To install dependencies, run `npm install`
- To start the demos run `npm start`
- Visit "http://localhost:1234/<demo-name.html>"
```

I will copy this as well to the main README.md.

**Comment by gkjohnson** [comment]
Assuming this can be closed, now, with #219 - let me know if you want to reopen it.
